tests/ctests/AleTest.cpp

```diff
@@ -233,23 +235,40 @@ TEST(VelocityCoeffTest, InvalidInput) {
 }
 
 
-TEST(LinearInterpTest, ExmapleGetRotation) {
+TEST(RotationInterpTest, ExmapleGetRotation) {
   // simple test, only checks if API hit correctly and output is normalized
   vector<double> times = {0,  1,  2, 3};
   vector<vector<double>> rots({{1,1,1,1}, {0,0,0,0}, {1,1,1,1}, {0,0,0,0}});
   vector<double> r = ale::getRotation(rots, times, 2, ale::linear);
+  Eigen::Quaterniond quat(r[0], r[1], r[2], r[3]);
 
-  ASSERT_NEAR(0.707107,  r[0], 0.000001);
-  EXPECT_DOUBLE_EQ(0,  r[1]);
-  ASSERT_NEAR(0.707107, r[2], 0.000001);
-  EXPECT_DOUBLE_EQ(0, r[3]);
+  EXPECT_DOUBLE_EQ(1, quat.norm());
 }
 
 
-TEST(LinearInterpTest, GetRotationDifferentCounts) {
+TEST(RotationInterpTest, GetRotationDifferentCounts) {
   // incorrect params
   vector<double> times = {0, 1, 2};
   vector<vector<double>> rots({{1,1,1,1}, {0,0,0,0}, {1,1,1,1}, {0,0,0,0}});
   EXPECT_THROW(ale::getRotation(rots, times, 2, ale::linear), invalid_argument);
 
 }
+
+
+TEST(AngularVelocityInterpTest, ExmapleGetRotation) {
+  vector<double> times = {0,  1};
+  vector<vector<double>> rots({{0,0}, {1,0}, {0,1}, {0,0}});
+  vector<double> av = ale::getAngularVelocity(rots, times, 0.5, ale::linear);
+
+  Eigen::Quaterniond quat(0, 1, 1, 0);
+  quat = quat.normalized();
+
+  Eigen::Quaterniond dQuat(0, -1, 1, 0);
+  dQuat = dQuat.normalized();
+
+  Eigen::Quaterniond avQuat = quat.conjugate() * dQuat;
```

Comment:
This test looks like it performs the steps in `getAngularVelocity` without calling the function. Why?

Reply:
I'll redo the math by hand so that we have a better comparison. It's still going to be a bit messy because of normalization.